components/tooltip/Tooltip.js

```diff
@@ -1,10 +1,10 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Box from '../box';
+import Transition from 'react-transition-group/Transition';
 import cx from 'classnames';
-import Portal from '../hoc/Portal';
+import { createPortal } from 'react-dom';
 import { getViewport } from '../utils/utils';
-import events from '../utils/events';
 import theme from './theme.css';
 
 const POSITION = {
```

Comment:
`SIZES` is plural, `POSITION` isn't? 🐜

Reply:
Valid point!

 :ok: by @driesd on 27/11/2017 14:21:35